MIGRATION.md

```diff
@@ -1,5 +1,13 @@
 # Deprecations in 7.x
 
+You can use [@sentry/migr8](https://github.com/getsentry/sentry-migr8) to automatically update your SDK usage and fix most deprecations:
```

Comment:
Should we link to the npm package instead? The npm page always has the correct docs and stuff.

Reply:
sounds like a good idea! 

```suggestion
You can use [@sentry/migr8](https://www.npmjs.com/package/@sentry/migr8) to automatically update your SDK usage and fix most deprecations:
```